lib/definition.js

```diff
@@ -280,6 +349,38 @@ module.exports = class Definition {
       })
     }
   }
+
+  _validateList() {
+    assert(this._list_type, `Invalid Env Config (${this._name}): Missing .type()`)
+    const is_array = Array.isArray(this._value)
+    if (this._required && (!is_array || !this._value.length)) {
+      throw new MissingEnvError({
+        type: 'list'
+      , name: this._env
+      })
+    }
+
+    if (!is_array) return
```

Comment:
should this resolve to an empty array when no value is provided?

Reply:
This case is, from what I can tell when the environment variable isn't set. In that situation, all them kind of behave this way.

if you want that, you can set a `default` or make it required.

Reply:
that's fair, setting a default is reasonable